eth/tracers/internal/tracetest/firehose/helpers_test.go

```diff
@@ -29,16 +29,17 @@ type firehoseInitLine struct {
 
 type firehoseBlockLines []firehoseBlockLine
 
-func newFirehoseTestTracer(t *testing.T, model tracingModel) (*tracers.Firehose, *tracing.Hooks, func()) {
+func newFirehoseTestTracer(t *testing.T, model tracingModel, concurrentBlockFlushing bool) (*tracers.Firehose, *tracing.Hooks, func()) {
```

Comment:
We should pass `FirehoseConfig` here I think and it would be inlined in the config